app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    list_of_args = {}
+
+    def inner(*args, **kwargs) -> Callable:
```

Comment:
You do not return `Callable` in the `inner`, it could be `Any` because you cannot foresee the result of the function execution you return here.